projects/eval-adxl313z/src/platform/stm32/main.c

```diff
@@ -62,6 +66,12 @@ int main()
 
 	stm32_init();
 
+#ifdef IIO_EXAMPLE
+	ret = iio_example_main();
+	if (ret < 0)
```

Comment:
nitpick: if (ret) syntax is being used on Linux, would be nice to use the same syntax